stdlib/2.7/socket.pyi

```diff
@@ -22,7 +22,7 @@ SOCK_CLOEXEC = 0
 SOCK_NONBLOCK = 0
 SOMAXCONN = 0
 has_ipv6 = False
-_GLOBAL_DEFAULT_TIMEOUT = 0.0
+_GLOBAL_DEFAULT_TIMEOUT = ...  # type: Any
```

Comment:
I'm not sure -- From the docs I know it could be None, so a safe bet would be Optional[Union[int, float]], but it's initially set to "object()", so...